tests/src/test/scala/cats/tests/KleisliTests.scala

```diff
@@ -93,6 +93,12 @@ class KleisliTests extends CatsSuite {
     checkAll("SemigroupK[Lambda[A => Kleisli[Option, A, A]]]", SerializableTests.serializable(kleisliSemigroupK))
   }
 
+  {
+    implicit val kleisliContravariant = Kleisli.kleisliContravariant[Option, Int]
```

Comment:
No, I've just checked and it works fine without. I was just blindly following the pattern in the rest of the test suite. Happy to push up without if that's preferred.